js/src/pages/settings/product-catalog/index.js

```diff
@@ -121,6 +136,7 @@ const ProductCatalog = () => {
 						variant="secondary"
 						onClick={ handleOnGenerateCsvClick }
 						loading={ exportInProgress }
+						eventName="sfw_regenerate_csv_button_click"
```

Comment:
@Sidsector9 It'll be good to include the context as well, which can be done via the `eventProps` prop with the following object as value:
```
{
	context: 'settings',
}
```